frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -44,7 +33,8 @@ export const OCSStorageClassDropdown: React.FC<OCSStorageClassDropdownProps> = (
 
   const handleStorageClass = (sc: K8sResourceKind) => {
     const name = getName(sc);
-    onChange(name);
+    const provisioner: string = sc?.provisioner;
+    onChange(name, provisioner);
```

Comment:
This would cause problems in the node-list page where you're directly applying this to the setState and it only expects one argument and not two.

Reply:
`handleStorageClass` is called on `onChange` method and it accepts 2 params. Look carefully.

Reply:
Please look at https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx#L329 
It's directly connected to the dispatcher which takes only 1 prop. This is an issue.

Reply:
It has been changed with this PR. See this - https://github.com/openshift/console/pull/5016/files#diff-10211f95caf12c0c4e086214ec83e140R326